landing-page/types/alces-branding/content/styles/branding.css

```diff
@@ -49,9 +49,6 @@ body {
 }
 
 .navbar.brandbar .navbar-nav-container,
-.navbar.productbar .navbar-nav-container {
-    justify-content: end;
-}
```

Comment:
This doesn't look right.  We still need the `justify-content: end;`.  Without it we end up with the flight direct branding being aligned to the left instead of the right.

![image](https://user-images.githubusercontent.com/287050/163437230-7b88e4ab-229f-423d-83f8-0646abbe79c7.png)

Reply:
Didn't notice that lines 51 and 52 were getting the same block; woops.